index.html

```diff
@@ -40,23 +41,38 @@ <h2>New Entry</h2>
             <img class="img-size" src="images/placeholder-image-square.jpg" alt="placeholder image">
           </div>
           <div class="column-half">
-            <div class="label"><label for="title">Title</label></div>
-            <div><input id="title" class="input" type="text" name="title" required="required"></div>
-            <div class="label"><label for="photoURL">Photo URL</label></div>
-            <div><input id="photoURL" class="input" type="text" name="photoURL" required="required"></div>
+            <div class="label font-weight-six"><label for="title">Title</label></div>
+            <div><input id="title" class="input font-weight-four" type="text" name="title" required="required"></div>
+            <div class="label font-weight-six"><label for="photoURL">Photo URL</label></div>
+            <div><input id="photoURL" class="input font-weight-four" type="text" name="photoURL" required="required"></div>
           </div>
         </div>
         <div class="row">
           <div class="column-full">
-            <div class="label"><label for="notes">Notes</label></div>
-            <div><textarea id="notes" class="text-area" name="notes" required="required"></textarea></div>
+            <div class="label font-weight-six"><label for="notes">Notes</label></div>
+            <div><textarea id="notes" class="text-area font-weight-four" name="notes" required="required"></textarea></div>
           </div>
           <div class="column-full flex-end">
             <button class="button">SAVE</button>
           </div>
         </div>
       </form>
     </div>
+    <div class="container" data-view="entries">
+      <div class="row">
+        <div class="column-full flex-wrap black">
+          <h2>Entries</h2>
+          <button class="button new-button">NEW</button>
+        </div>
+      </div>
+      <ul id="entry-list">
+        <li id="example" class="row">
+          <div class="column-full text-center">
+            <p class="p-font">No entries have been recorded.</p>
+          </div>
+        </li>
```

Comment:
You shouldn't make this an `li`, since it isn't an entry. I would just put this element above the `ul` as a sibling.